index.html

```diff
@@ -234,6 +235,7 @@
       openmct.install(openmct.plugins.Timelist());
       openmct.install(openmct.plugins.BarChart());
       openmct.install(openmct.plugins.ScatterPlot());
+      openmct.install(openmct.plugins.EventTimestripPlugin(timeLinePlugin.extendedLinesBus));
```

Comment:
we need to decide if we want this to be the default plugin in core. Traditionally, we don't add new plugins for e2e tests and run the `addInitScript` to add the plugin at runtime.